### DISCUSSION TITLE
Anybody have a windows 11 start button replica?

### DISCUSSION THREAD

**Discussion by klepp0906** [Open-Shell/Open-Shell-Menu#1275] [General]
Since this is not accessible without using the custom start button option, hoping someone has something that mirrors/mimics microsofts for now.

with the resolution/scaling I have set up the built in button skins dont cover the default button and thats obviously a disaster :P

**Comment by gandhule90** [comment]
not mine, but this is the windows 10x start button: https://www.deviantart.com/joaofernandojfmx/art/Start-Button-Windows-11-10x-832542022  it looks like windows 11 start button replica for open shell and classic shell but isn't the full same

**Comment by klepp0906** [comment]
yea it looks to be angled as opposed to flat.  still better than the alternative and better what i was able to drudge up on short notice.  thank you for linking it <3

**Comment by gandhule90** [comment]
yes hope somebody comes with the flat one and with the perfect size..

**Comment by s-eremin** [comment]
Having found no publicly available solution, I just made a decent Windows 11 _Start button_ replica for myself. I designed it for a Full HD (1920×1080) display with a scale of 125% (for a laptop). This replica will probably look suboptimal in other resolutions and scales. To properly use it in Open-Shell, please set _Button look_ to _Classic button_, set _Button icon_ to [_Windows 11 Start 56x56 (32x32).ico_](https://github.com/Open-Shell/Open-Shell-Menu/files/7274552/Windows.11.Start.56x56.32x32.zip), set _Icon size_ to _56_, set _Button text_ to an empty string. For a Full HD display with a 100% scale, setting _Icon size_ to _44_ does the trick, although in this case the icon has to be resized by software and may look just a little blurry.

**Comment by s-eremin** [comment]
And in case someone wishes to play with the image on his/her own to fit it best to a specific display resolution and scale, I managed to extract the button in a ["huge" 115×115 px resolution](https://user-images.githubusercontent.com/91852728/135778925-ee56372f-b271-4837-87cc-ccffbe507079.png).

**Comment by bonzibudd** [comment]
Just for reference, the latest nightly builds now map to the Windows 11 Start button.
